README.md

```diff
@@ -2,10 +2,10 @@
 <img width=50% src="https://evalml-web-images.s3.amazonaws.com/evalml_horizontal.svg" alt="Featuretools" />
 </p>
 
-[![CircleCI](https://circleci.com/gh/alteryx/evalml/tree/main.svg?style=svg&circle-token=9e0ce5e5f2db05f96fe92238fcde6d13963188b6)](https://circleci.com/gh/alteryx/evalml/tree/main)
-[![codecov](https://codecov.io/gh/alteryx/evalml/branch/main/graph/badge.svg?token=JDc0Ib7kYL)](https://codecov.io/gh/alteryx/evalml)
-[![PyPI version](https://badge.fury.io/py/evalml.svg?maxAge=2592000)](https://badge.fury.io/py/evalml)
-[![PyPI - Downloads](https://img.shields.io/pypi/dm/evalml.svg)](https://pypistats.org/packages/evalml)
+[![GitHub Actions](https://github.com/alteryx/evalml/actions/workflows/linux_unit_tests.yml/badge.svg)](https://github.com/alteryx/evalml/actions/workflows/linux_unit_tests.yml)
```

Comment:
I think we should set `?branch=main` so that it only picks up the main unit tests?

Reply:
Nevermind, picks up `main` by default!

Reply:
Good call, by default they include `main` but I just updated to be safe
https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-branch-parameter